src/_codux/boards/calendar-date/calendar-date.board.tsx

```diff
@@ -0,0 +1,24 @@
+import { ContentSlot, createBoard } from '@wixc3/react-board';
+import { CalendarDate } from '../../../components/calendar-date/calendar-date';
+import styles from './calendar-date.board.module.scss';
+import { ComponentWrapper } from '/src/_codux/board-wrappers/component-wrapper';
+
+export default createBoard({
+    name: 'CalendarDate',
+    Board: () => (
+        <ComponentWrapper settings={{}}>
+            <ContentSlot>
+                <div className={styles.container}>
+                    <CalendarDate isAvailable={true} date={9} isSelected={false} />
+                    <CalendarDate isAvailable={true} date={9} isSelected={true} />
+                    <CalendarDate isAvailable={false} date={9} isSelected={false} />
+                    <CalendarDate isAvailable={false} date={9} isSelected={true} />
```

Comment:
How about making `isAvailable` and `isSelected` optional, and then you can do something like this:
```js
<CalendarDate date={9} isAvailable />
<CalendarDate date={9} isAvailable isSelected />
<CalendarDate date={9} />
<CalendarDate date={9} isSelected />
```

Looks cleaner IMO.